pkg/agent/qrm-plugins/cpu/dynamicpolicy/policy.go

```diff
@@ -629,7 +663,7 @@ func (p *DynamicPolicy) GetTopologyHints(ctx context.Context,
 		return nil, fmt.Errorf("getReqQuantityFromResourceReq failed with error: %v", err)
 	}
 
-	general.InfoS("called",
+	general.InfoS("GetTopologyHints called",
```

Comment:
since general logging framwork will add function name automatically, there is no need to add GetTopologyHints in log str again.